cargo-dist/src/config.rs

```diff
@@ -845,6 +854,30 @@ impl std::fmt::Display for InstallerStyle {
     }
 }
 
+/// When to create GitHub releases
+#[derive(Clone, Copy, Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub enum GithubReleasePhase {
+    /// Release position depends on whether axo releases is enabled
+    #[default]
+    Auto,
+    /// Create release during the "host" stage, before npm and Homebrew
+    Host,
+    /// Create release during the "announce" stage, after all publish jobs
+    Announce,
+}
+
+impl std::fmt::Display for GithubReleasePhase {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let string = match self {
+            GithubReleasePhase::Auto => "auto",
```

Comment:
It would be more conventional for us to just use None to capture this but since it's a global setting this is fine, I suppose.

Reply:
Yeah I thought about this for a bit. From a user-facing config side, I always appreciate being able to explicitly specify a setting and still have the "auto" setting be an option, just for clarity.